test/dynamo/test_dynamo.py

```diff
@@ -18,6 +18,21 @@
 import test_utils
 
 
+class DynamoInPlaceTest(unittest.TestCase):
+
+  def inplace_update(self, a):
+    a += 1
+    return a
+
+  def test_inplace_update_correctness(self):
```

Comment:
nit: should we be adding compile/execute metrics checks here?

Reply:
good catch, let me add that in a follow up pr.